### DISCUSSION TITLE
Unable to create LXC container for AdGuard Home - unsupported debian version '12.2'

### DISCUSSION THREAD

**Discussion by Sr-psycho** [tteck/Proxmox#2037] [Help Needed] [answered]
Greetings! I tried to install a container with AdGuard home on proxmox ver. 7.4-3. But the script ends with the error "unable to create CT ХХХ - unsupported debian version '12.2' A problem occurs while trying to create container.: numeric argument required"

And this message appears on the screen after running the script:
Proxmox VE 7 Detected
You are currently using Proxmox VE 7, refrain from
creating Debian 12 LXCs due to differences in locale
parameters.
Default distribution for Adguard LXC is debian 12

Please help me how to solve this problem.

**Comment by tteck** [comment, marked as answer]
Upgrade to Proxmox VE 8

**Reply by tteck** [reply]
Or, use "Advanced settings" and select Debian 11
